packages/storage/amplify_storage_s3_dart/test/amplify_storage_s3_dart_test.dart

```diff
@@ -140,14 +141,16 @@ void main() {
       );
 
       setUpAll(() {
-        registerFallbackValue(const S3ListOptions());
+        registerFallbackValue(
+          const StorageListOptions<S3ListPluginOptions>(
+            accessLevel: testDefaultStorageAccessLevel,
```

Comment:
nit: this default value can be omitted, `registerFallbackValue` just need a type reference here and the actual properties are not used.